mwdb/web/src/components/UserProfile.js

```diff
@@ -68,9 +79,27 @@ class UserProfile extends Component {
                         </tr>
                     </thead>
                     <tbody>
-                        <tr>
+                        <tr className="flickerable">
+                            <td>Login</td>
+                            <td>
+                                {this.state.profile.login}
+                                <span className="ml-2">
+                                    <i data-toggle="tooltip" title="Click to search for user samples">
+                                        <Link to={makeSearchLink("uploader", this.state.profile.login)} style={{color:'black'}}>
```

Comment:
Maybe separate button for that? It's a bit hidden and confusing that it's in the same place as "Copy to clipboard"